sqlite/xen.x86_64.cfg

```
@@ -0,0 +1,7 @@
+name          = "sqlite"
+vcpus         = "1"
+kernel        = "./workdir/build/sqlite_xen-x86_64"
+ramdisk       = "./initrd.cpio"
+cmdline       = "sqlite_xen-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- sqlite3 -interactive"
```

```suggestion
cmdline       = "sqlite_xen-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /chinook.db 'SELECT * FROM Album LIMIT 10'"
```

You want this to be default, right?